### PR TITLE
[ja] Localize Loosely Coupled Architecture for Japanese

### PR DESCRIPTION
**Pull request by bashi8128**
### Describe your changes
Localize Loosely Coupled Architecture for Japanese.

### Related issue number or link (ex: `resolves #issue-number`)
#2446

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by naonishijima — APPROVED** [review]
LGTM

File: content/ja/loosely-coupled-architecture.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: 粗結合アーキテクチャ
```
Comment (kaitoii11) [inline content/ja/loosely-coupled-architecture.md:2]:
```suggestion
title: 疎結合アーキテクチャ
```

File: content/ja/loosely-coupled-architecture.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+status: Completed
+category: プロパティ
+tags: ["基礎", "アーキテクチャ", "プロパティ"]
+---
+
+粗結合アーキテクチャは、アプリケーションの構成要素それぞれが互いに独立して構築されるようなアーキテクチャです
```
Comment (kaitoii11) [inline content/ja/loosely-coupled-architecture.md:8]:
```suggestion
疎結合アーキテクチャは、アプリケーションの構成要素それぞれが互いに独立して構築されるようなアーキテクチャです
```

File: content/ja/loosely-coupled-architecture.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+(その逆は[密結合アーキテクチャ](/ja/tightly-coupled-architectures/)と呼ばれます)。
+[マイクロサービス](/ja/microservice/)とよく呼ばれる個々の構成要素は、その他多くのサービスから利用できるような方法で、特定の機能を果たすよう構築されます。
+粗結合アーキテクチャは密結合アーキテクチャより一般的に実装が遅くなりますが、特にアプリケーションがスケールする際に多くの利点があります。
```
Comment (kaitoii11) [inline content/ja/loosely-coupled-architecture.md:11]:
```suggestion
疎結合アーキテクチャは密結合アーキテクチャより一般的に実装が遅くなりますが、特にアプリケーションがスケールする際に多くの利点があります。
```

File: content/ja/loosely-coupled-architecture.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+アプリケーションが粗結合だと、チームは独立して機能開発、デプロイ、スケールすることが可能です。
```
Comment (kaitoii11) [inline content/ja/loosely-coupled-architecture.md:13]:
```suggestion
アプリケーションが疎結合だと、チームは独立して機能開発、デプロイ、スケールすることが可能です。
```

**Review by kaitoii11 — CHANGES_REQUESTED** [review]

**Comment by bashi8128** [comment]
あらゆるところで誤字を生んでいました。
「粗」を「疎」に修正しました。

**Review by kaitoii11 — APPROVED** [review]
LGTM

**Pull request merged**